### PR TITLE
DOCS-7489 rtfctl command list

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

**Review by sarjeet2013 — APPROVED** [review]
LGTM

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 1]
```diff
@@ -32,15 +32,47 @@ sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/lat
...
+| help | Display help for a command.
+| memory | Display the JVM memory usage in a pod.
+| package | Create a debugging information package for an application.
+| ping | Test network connectivity.
+| report | Generate diagnostics report.
+| restart | Restart an application.
+| restore | Restore local Runtime Fabric state from a backup.
+| status | Display status of Runtime Fabric.
+| test | Test Runtime Fabric environment.
+| threaddump | Display the JVM thread dump for an application.
```
Comment (kbostrom) [inline modules/ROOT/pages/install-rtfctl.adoc]: Why include an article here (the) when none of the other descriptions use them?

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 2]
```diff
@@ -32,15 +32,47 @@ sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/lat
...
+| version | Display version information.
+|=== 
 
-For more information on a specific command, run `rtfctl <command> -h`.
+[%header%autowidth.spread,cols="a,a"]
+|===
+|Flag |Description
+| -h, --help | Display `rtfctl` help information.
+| -n, --namespace string | If present, display the namespace scope.
+| --pod string | If present, display the pod name.
```
Comment (kbostrom) [inline modules/ROOT/pages/install-rtfctl.adoc]: articles here, too
Just inconsistent

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 3]
```diff
@@ -32,15 +32,47 @@ sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/lat
...
+|===
 
 == Usage
 
-* Install a Mule license
+* To list all supported commands, run `rtfctl -h`. 
+
+* For more information on a specific command, run `rtfctl <command> -h`.
```
Comment (kbostrom) [inline modules/ROOT/pages/install-rtfctl.adoc]:
```suggestion
* For more information about a specific command, run `rtfctl <command> -h`.
```

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 4]
```diff
@@ -32,15 +32,47 @@ sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/lat
...
-For more information on a specific command, run `rtfctl <command> -h`.
+[%header%autowidth.spread,cols="a,a"]
+|===
+|Flag |Description
+| -h, --help | Display `rtfctl` help information.
+| -n, --namespace string | If present, display the namespace scope.
+| --pod string | If present, display the pod name.
+|===
 
 == Usage
```
Comment (kbostrom) [inline modules/ROOT/pages/install-rtfctl.adoc:70]: These sound like steps more than list items.

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 5]
```diff
@@ -32,15 +32,47 @@ sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/lat
 sudo chmod +x rtfctl
 ----
 
-== List Supported Commands
+== Supported Commands
 
-To list all supported commands, run `rtfctl -h`. 
+[%header%autowidth.spread,cols="a,a"]
+|===
+|Command |Description
```
Comment (kbostrom) [inline modules/ROOT/pages/install-rtfctl.adoc:39]: Hmm...it always sort of bothers me when the entries in a column titled "Description" are verb-based rather than noun-based. We are not here telling the reader what to DO with the command; we are describing what the command does. I would reword each of these to "specifies," "applies," and so on.
Or I would change "Description" to imply a user action instead.

**Review by kbostrom — APPROVED** [review]
Done
